### PR TITLE
Get nginx container id from labelled container

### PR DESCRIPTION
**Pull request by emmetog**
Fixes https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion/issues/170

### CODE REVIEW

**Comment by emmetog** [comment]
Not tested 100% yet, when I test it in docker compose v3 I'll post a comment. Feel free to comment on this PR in the meantime.

**Comment by Tabrizian** [comment]
It's fully functional and completely compatible with docker compose v3. Thanks for your PR :+1:

File: README.md [hunk 1]
```diff
@@ -30,14 +30,15 @@ $ docker run -d -p 80:80 -p 443:443 \
     -v /etc/nginx/vhost.d \
     -v /usr/share/nginx/html \
     -v /var/run/docker.sock:/tmp/docker.sock:ro \
+    --label jrcs.nginx_letsencrypt_companion.is_proxy=true \
```
Comment (teohhanhui) [inline README.md]: It should use a fully qualified domain name, i.e. `com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy=true`

Comment (JrCs) [inline README.md]: Yes it will better to use a fully qualified domain name.
@emmetog can you make the changes ?

File: README.md [hunk 2]
```diff
@@ -30,14 +30,15 @@ $ docker run -d -p 80:80 -p 443:443 \
     -v /etc/nginx/vhost.d \
     -v /usr/share/nginx/html \
     -v /var/run/docker.sock:/tmp/docker.sock:ro \
+    --label com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy=true \
```
Comment (teohhanhui) [inline README.md]: There's no need for `=true`. Just a plain label is enough. It doesn't have to be a key-value pair.

**Pull request merged**